src/platform/unix.rs

```diff
@@ -78,6 +78,8 @@ pub trait WindowExtUnix {
 
   /// Whether to show the window icon in the taskbar or not.
   fn set_skip_taskbar(&self, skip: bool) -> Result<(), ExternalError>;
+
+  fn set_badge_count(&self, count: Option<i64>, desktop_filename: Option<String>);
```

Comment:
This should be on `EventLoopWindowTargetExtUnix`as well

Reply:
Done